### DISCUSSION TITLE
Will this work on v70.00.021?

### DISCUSSION THREAD

**Discussion by NabeelUppel** [mzd-evo/mzd-connect-1-root#6] [Q&A]
Hi! 
I would like to try this on my car that's running 70.00.021. I haven't installed anything before this point. 

Will this work?
Thanks in advance

**Comment by pjeantaud** [comment]
Bello,

I've no clue if it works with this version, it might probably.
Mine is 74.00.324 EUN and it works like a charm.

**Reply by sydragos** [reply]
It will work, may need to adjust the scripts for version checking. I used it to customize my v70 for the Fiat 124. Will use again once I update to v74
